### ISSUE TITLE
Для отладки серверного модуля помещать его во внешнюю обработку

### ISSUE THREAD

**Issue by DoublesunRUS** [DoublesunRUS/ru.capralow.dt.unit.launcher#5]
Привет. Идея для твоего плагина. 
Возможность перенаправить нужный мне модуль во внешнюю epf. Чтобы вызов каждой процедуры перенаправлялся во внешнюю обработку. Тогда можно будет редактировать модуль конфигурации без перезапуска предприятия.

И соответственно потом нужна возможность забрать код из epf обратно в модуль.

**Comment by DoublesunRUS** [comment]
Ну так просто вряд ли получится. Это нужно сделать расширение, в которое добавляешь общий модуль. В этом общем модуле все процедуры методы переопределяешь на вызов из внешней обработки.
А потом обратное действие. Надо код из обработки забрать в модуль.
В принципе наверное можно что-то такое придумать.

**Comment by Pr-Mex** [comment]
Да. Это позволит сильно упросить жизнь на больших конфигурациях, т.к. позволит и редактировать модули без перезапуска базы, так и редактировать тесты без перезапуска базы.
